TikTokTrainer/Camera/CameraModel.swift

```diff
@@ -318,6 +318,11 @@ extension CameraModel: AVCaptureVideoDataOutputSampleBufferDelegate {
 
     func captureOutput(_ output: AVCaptureOutput, didOutput sampleBuffer: CMSampleBuffer, from connection: AVCaptureConnection) {
         if let pixelBuffer = sampleBuffer.imageBuffer {
+            if self.currentOrientation == AVCaptureDevice.Position.front {
+                connection.isVideoMirrored = true
+            } else {
+                connection.isVideoMirrored = false
+            }
```

Comment:
This can be replaced with just connection.isVideoMirrored = self.currentOrientation == .front

Reply:
Move this to flip feed button because this is probably expensive to change every single frame